### PR TITLE
Feature: Show tagline in movie details

### PR DESCRIPTION
**Pull request by wutschel**
## Description

This PR adds the tagline to the movie details screen. This works for all ways to enter movie details in the movie menu (movies, genres, movies sets, recently added, playlists).

Screenshot: https://ibb.co/XWHP4wt

## Summary for release notes

Feature: Show tagline in movie details

### CODE REVIEW

**Comment by wutschel** [comment]
Not for 1.14, so made it draft for now.

**Comment by wutschel** [comment]
Made this "ready for review" after we now released 1.14 and can bring in changes.

**Comment by wutschel** [comment]
Converted to draft as this conflicts with #1004, which needs priority.

**Comment by wutschel** [comment]
Rebased to master, resolved conflict and ready-for-review.

File: XBMC Remote/ShowInfoViewController.h [hunk 1]
```diff
@@ -23,25 +23,23 @@
...
+    UILabel *mainLabel5;
+    UILabel *parentalRatingMainLabel;
+    UILabel *castMainLabel;
+    
+    UILabel *subLabel0;
+    UILabel *subLabel1;
+    UILabel *subLabel2;
+    UILabel *subLabel3;
+    UILabel *subLabel4;
+    UILabel *subLabel5;
```
Comment (kambala-decapitator) [inline XBMC Remote/ShowInfoViewController.h:41]: why not giving proper names instead of labelX?

File: XBMC Remote/ShowInfoViewController.m [hunk 2]
```diff
@@ -1026,6 +1045,34 @@ - (CGFloat)layoutStars:(CGFloat)offset {
...
+    label.font = [UIFont systemFontOfSize:12];
+    label.textAlignment = NSTextAlignmentLeft;
+    label.lineBreakMode = NSLineBreakByTruncatingTail;
+    label.numberOfLines = 1;
+    [scrollView addSubview:label];
+    return label;
+}
+
+- (UILabel*)createSubLabel {
+    CGRect defaultFrame = CGRectMake(LABEL_PADDING, 0, scrollView.frame.size.width - 2 * LABEL_PADDING, LABEL_HEIGHT);
```
Comment (kambala-decapitator) [inline XBMC Remote/ShowInfoViewController.m:1072]: you could call createMainLabel and only adjust a few properties on the returned object

**Review by kambala-decapitator — APPROVED** [review]

File: XBMC Remote/ShowInfoViewController.h (hunk 1 above)
Comment (wutschel) [reply XBMC Remote/ShowInfoViewController.h:41]: All these labels are multi-purpose, the content depends on the item for which the details are shown. Simple numbering scheme seemed the best approach.

**Pull request merged**